experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -93,6 +91,9 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 	private readonly editManager: EditManager<TEditor, TChange, ChangeFamily<TEditor, TChange>>;
 	private readonly summarizables: readonly Summarizable[];
 
+	/** Iff false, calls to `submitOp` will have no effect */
+	private submitOps = true;
```

Comment:
I don't like having to have this here in order to prevent submitting ops when applying stashed ops, but I can't think of anything better